docs/backend-requests/resources/session-tokens.mdx

```diff
@@ -32,6 +34,22 @@ If you would like to add custom claims to your session token, you can [customize
 
 You can also create custom tokens using a [JWT template](/docs/backend-requests/making/jwt-templates).
 
+## Size limitations
+
+The Clerk session token is stored in a cookie. All modern browsers [limit the maximum size of a cookie to 4kb](https://datatracker.ietf.org/doc/html/rfc2109#section-6.3). Exceeding the 4kb max size can have adverse effects, including a possible infinite redirect loop for users who exceed this size in Next.js applications.
+
+A session token with the [default session claims](#default-session-claims) will not run into this issue, as this configuration produces a cookie significantly smaller than 4kb. The size limitation only becomes a consideration when implementing a [custom session token](/docs/backend-requests/making/custom-session-token) or creating a [custom JWT template](/docs/backend-requests/making/jwt-templates). In these cases, it's recommended to move particularly large claims out of the token and fetch these using a separate API call from your backend.
+
+Large claims to be cautious about:
+
+- `user.organizations`
+- `user.public_metadata`
+- `user.unsafe_metadata`
+- `org.public_metadata`
+- `org_membership.public_metadata`
+
+If you are including any of these claims in your JWT, use extra caution around how much data is stored in them to avoid exceeding the size limit.
```

Comment:
```suggestion
If you are including any of these claims in your token, use extra caution around how much data is stored in them to avoid exceeding the size limit.
```